demo/api/package.json

```diff
@@ -17,9 +17,6 @@
         "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
         "lint:tsc": "tsc --project ./tsconfig.lint.json",
         "lint:generated-files-not-modified": "$npm_execpath api-generator && git diff --exit-code HEAD -- src/**/generated",
-        "mikro-orm": "mikro-orm",
```

Comment:
This will be needed again in next due to MikroORM v6, we (i.e. I 😁)  should consider this when merging main into next.